storage/storage-proto/src/lib.rs

```diff
@@ -552,7 +553,7 @@ impl From<TreeState> for crate::proto::storage::TreeState {
 }
 
 /// Helper to construct and parse [`proto::storage::StartupInfo`]
-#[derive(Clone, Debug, Eq, PartialEq)]
+#[derive(Clone, Debug, Eq, PartialEq, Serialize, Deserialize)]
 pub struct StartupInfo {
```

Comment:
you would be my hero if you pushed out a PR that moves this into storage-interface or something...

Reply:
next, next, next.